### ISSUE TITLE
Link to change log on RTD docs is broken

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbem#175]
The main page on RTD contains a change log section. The link is broken, even though the NEWS.md file is there.

**Comment by andy-maier** [comment]
It turns out the NEWS.md file is only there after a local `make builddoc`, but not on RTD. Reason being that the make rule copies the file. On RTD, Sphinx is started directly by their logic, so the make rule does not get invoked.

The only solution I see is to reference the NEWS file not by a link, but by "official" means so that Sphinx processes the file. These means are:
* include statement
* toctree statement
In all cases, the headings in the NEWS file merge into the headings of the rest of the document, so the heading level in the NEWS file needs to be demoted.

Basically, the main decision is whether to change the documentation into a multi-page document.
